docs/meshstack.metadata-tags.md

```diff
@@ -22,168 +25,42 @@ such metadata include:
+You will be prompted with a new page where you can enter a lot of information. We will briefly explain what each fields means and why you should use it.
+
+> Hint: after entering a name, you can view the tag input as if it were shown to the end user. You can do this on the right-side of the screen where it says 'Tag Preview'.
+> This can help you get an idea of how it will look when end-users fill in the tag value.
+
+- **Display Name**: This is the text that will be shown above the input field which the user reads. Note that is **not** the name of the tag itself.
+- **Name**: This is the actual name of the tag as it will be displayed in the green bubble. It is also the name that is used for uniquely identifying the tag and when exporting it or when [using it within a landing zone](#exposing-tags-via-landing-zones). This name cannot be changed later (at the moment). We recommend using a name that is similar or the same as the Display Name. Keep in mind that the use of spaces is not allowed since the tag name is used for technical purposes.
+- **Description (optional)**: This is the description of the tag that will be displayed in small grey text above the tag input. It is useful for giving extra information to the user when the (display) name of the tag is not explanatory enough.
+- **Tag Type**: This is the data type of the tag and also decides what kind of input the user will see. It will depend what you will fill in here depending on your needs. You can see the actual input in the Tag Preview on the right if you are curious how it looks.
+- **Regular Expression (optional)**: This is only available for `e-mail` and `string` tag types and can be used to enforce that users enter a certain data format. Also make sure to mention this in the description to help users what to enter.
+- **Default Value (optional)**: This can be used to give a default value to the user for contextual help or for filling in a default.
+- **Mandatory**: If a tag is mandatory, it means the user cannot create a new meshObject without first filling in this tag value.
+- **Restricted**: If a tag is restricted, it means only Partner users can enter this tag value. The input will also not be shown to the end-user when creating a new meshObject. This is useful if an organization doesn't want users to edit these values in self-service. For example, an organization can use restricted tags to implement a "segregation of duty" control to ensure that a partner user confirmed a project's data classification.
+- **Replication**: (note: not supported for meshLandingZones) If a tag has replication enabled, it will be used to tag an actual tenant in the cloud platform.
+  - **Replication Tag Key**: Although we recommend using the exact same value as **Name** here, it possible to use a slightly different tag key when it is used for replication. Also, keep in mind that the tag key will be prefixed, depending on what is configured for [tenant tags](#tags-in-cloud-tenants).
```

Comment:
d: I think also tag sanitization will be applied for this field right? Maybe we should add a sentence that mentions this, to make users aware, that these could differ in the various clouds.